### PR TITLE
spicetify-cli 2.27.1 (new formula)

### PR DESCRIPTION
**Pull request by razvanazamfirei**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/s/spicetify-cli.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "globals.d.ts",
+        "jsHelper",
+        "Themes",
+      ]
+      bin.install_symlink libexec/"spicetify"
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/spicetify -v")
```
Comment (chenrui333) [inline Formula/s/spicetify-cli.rb:31]: need a more substantial test

File: Formula/s/spicetify-cli.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      ]
+      bin.install_symlink libexec/"spicetify"
+    end
+  end
+
+  test do
+    spotify_folder = testpath/"com.spotify.Client"
+    pref_file = spotify_folder/"com.spotify.client.plist"
+    mkdir_p spotify_folder
+    touch pref_file
```
Comment (chenrui333) [inline Formula/s/spicetify-cli.rb:34]:
```suggestion
```

**Review by chenrui333 — APPROVED** [review]
almost good, just one comment

File: Formula/s/spicetify-cli.rb (hunk 2 above)
Comment (razvanazamfirei) [reply Formula/s/spicetify-cli.rb:34]: @chenrui333, the files need to exist otherwise `spicetify` will error out.

Comment (chenrui333) [reply Formula/s/spicetify-cli.rb:34]: I see, sounds good then.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**